modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionServiceImpl.java

```diff
@@ -46,6 +46,23 @@
 public class ObjectDefinitionServiceImpl
 	extends ObjectDefinitionServiceBaseImpl {
 
+	@Override
+	public ObjectDefinition addCustomObjectDefinition(
+			Boolean enableComments, Map<Locale, String> labelMap, String name,
+			String panelAppOrder, String panelCategoryKey,
+			Map<Locale, String> pluralLabelMap, String scope,
+			String storageType, List<ObjectField> objectFields)
+		throws PortalException {
```

Comment:
@TarcianaBarros please change the existing `addCustomObjectDefinition` instead of creating a new one.. Thanks!